OpenRA.Game/FieldLoader.cs

```diff
@@ -135,6 +135,14 @@ public static T Load<T>(MiniYaml y) where T : new()
 			return t;
 		}
 
+		public static T LoadField<T>(Dictionary<string, MiniYaml> d, string key, T fallback)
```

Comment:
This isn't really appropriate to live in `FieldLoader`. In the context of FieldLoader, the term "Field" refers specifically to the C# Field (i.e. member/property) types, whereas the sequence parsers use "Field" colloquially to mean "something in a yaml node that is loaded into a field using FieldLoader".

Mixing the two here adds confusion.

Reply:
Yes. Removed the change.